### DISCUSSION TITLE
Eleventy command not found

### DISCUSSION THREAD

**Discussion by matto2** [11ty/eleventy#2719] [Q&A]
I'm having a weird issue: This command works fine: `npx eleventy --serve ` but this command is not working: `eleventy -v` (result is zsh: command not found: eleventy) Any ideas?

 I'm on a new (to me) M1 laptop but not sure that is the cause - my older laptop did not have this issue.

**Comment by pdehaan** [comment]
If you don't have 11ty installed locally already, <kbd>npx eleventy</kbd> will have some possibly unexpected behavior of using the https://www.npmjs.com/package/eleventy package instead of the https://www.npmjs.com/package/@11ty/eleventy package.

Does <kbd>npx @11ty/eleventy -v</kbd> work?

**Reply by matto2** [reply]
@pdehaan, 'npx @11ty/eleventy -v' is not working

**Reply by pdehaan** [reply]
Sorry, my mistake, it is `--version` not `-v` alias:

```sh
npx @11ty/eleventy --version # 1.0.2
node --version # v18.12.1
npm --version # 8.19.2
```

**Reply by matto2** [reply]
This is what I get for those:
npx @11ty/eleventy --version
1.0.2
 node --version
v18.12.1
npm --version
9.2.0

**Reply by pdehaan** [reply]
Perfect, so it's working when you use the explicit <kbd>npx @11ty/eleventy</kbd>?
